### DISCUSSION TITLE
Pull shark not count correctly

### DISCUSSION THREAD

**Discussion by scruel** [community/community#19220] [Other Feature Feedback, Questions, & Ideas]
Also an old issue this haven't yet been solved
https://github.com/apache/rocketmq/commit/7b1ef8f33a7381056c04667aacb0445f2f0dd009
![image](https://user-images.githubusercontent.com/16933298/175181780-cf146c7c-90a2-488f-96c9-2b848e9fc86d.png)
![image](https://user-images.githubusercontent.com/16933298/175181788-0486c581-1afe-4c18-b7e7-f323128bc07d.png)

Also, many more others not counted:
https://github.com/EgorKulikov/idea-chelper
https://github.com/houbb/sensitive

**Comment by melvincwng** [comment]
@scruel, if I recall correctly, some backdated contributions aren't taken into account for the achievements.

Hope this clarifies.

**Reply by b-hayes** [reply]
nah its def not counting

**Comment by b-hayes** [comment]
Apparently, I first got this achievement in 2019. Since then I've had probably thousands of PRs merged and it counts only 3. I've probably done 3 in a single day before. 🤣

**Comment by baysideengineer** [comment]
Hi @scruel, please have a look at [this](https://github.com/github-community/community/discussions/18213#discussioncomment-2918994) or [this](https://github.com/github-community/community/discussions/18190#discussioncomment-2917831) answer. I think this will help you to understand how GitHub is counting pull requests. At the beginning this was very confusing for many users including me. Have a nice day.
